### PR TITLE
Attempt to use quadrature spaces for ODE-variables in mechanics model

### PR DESCRIPTION
**Pull request by finsberg**

### CODE REVIEW

File: src/simcardems/mechanics_model.py [hunk 1]
```diff
@@ -267,7 +296,7 @@ def solve(self):
         return super().solve()
 
     def update_lmbda_prev(self):
-        self.lmbda_prev.assign(dolfin.project(self.lmbda, self.lmbda_space))
+        self.lmbda_prev.vector()[:] = self.lmbda.vector()
```
Comment (cdaversin) [inline src/simcardems/mechanics_model.py:299]: We should maybe be a bit careful with that in parallel

File: sandbox/quadrature_space.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import dolfin
+import pulse
+
+
+quadrature_degree = 3
+dolfin.parameters["form_compiler"]["quadrature_degree"] = quadrature_degree
+dolfin.parameters["form_compiler"]["representation"] = "uflacs"
+
+
+def local_project(v, V, u=None):
```
Comment (cdaversin) [inline sandbox/quadrature_space.py:10]: Don't we have a local_project function in utils that we could use here ?

**Review by cdaversin — COMMENTED** [review]
Looks good to me! I just added minor comments

File: sandbox/quadrature_space.py (hunk 2 above)
Comment (finsberg) [reply sandbox/quadrature_space.py:10]: Yes we could. This was made as a toy example and I wanted it to be more of less self contained.

File: src/simcardems/mechanics_model.py (hunk 1 above)
Comment (finsberg) [reply src/simcardems/mechanics_model.py:299]: We also do it similarly for Zetas and Zetaw. What would be the best way of doing this in parallel?

**Pull request merged**